### PR TITLE
[apache_spark][node] Change naming convention of the node data stream

### PR DESCRIPTION
**Pull request by yug-elastic**
## What does this PR do?

- Refactor the `nodes` data stream to adjust its name to singular (`node`) as per the discussion [here](https://github.com/elastic/integrations/pull/2941#discussion_r846707836).

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/apache_spark directory.
- Run the following command to run tests. 
> elastic-package test

### CODE REVIEW

**Review by mtojek — CHANGES_REQUESTED** [review]
requested changes

File: packages/apache_spark/data_stream/node/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ processors:
       field: event.module
       value: apache_spark
   - remove:
-      field: apache_spark.nodes.mbean
+      field: jolokia
```
Comment (ruflin) [inline packages/apache_spark/data_stream/node/elasticsearch/ingest_pipeline/default.yml:21]: How is this change related? A bug found?

**Review by ruflin — APPROVED** [review]

File: packages/apache_spark/data_stream/node/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (yug-elastic) [reply packages/apache_spark/data_stream/node/elasticsearch/ingest_pipeline/default.yml:21]: Right, this change is unrelated. There was an empty field being generated in the sample event. We have made the same change in the other data streams as well and hence thought of shipping it here.

File: packages/apache_spark/changelog.yml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # newer versions go on top
-
+- version: "0.1.1"
+  changes:
+    - description: Refactor the nodes data stream to adjust its name to singular(node)
```
Comment (mtojek) [inline packages/apache_spark/changelog.yml:4]: nit-pick: singular`<space>`(node)

Comment (yug-elastic) [reply packages/apache_spark/changelog.yml]: Done

**Review by jsoriano — APPROVED** [review]
LGTM

**Pull request merged**